tests/DataSources/DbPrefill/ProjectsTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Tests\DataSources\DbPrefill;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\Website\DataSources\DataSource;
+use Doctrine\Website\DataSources\DbPrefill\Projects;
+use Doctrine\Website\Docs\RST\RSTLanguage;
+use Doctrine\Website\Git\Tag;
+use Doctrine\Website\Model\Project;
+use Doctrine\Website\Model\ProjectStats;
+use Doctrine\Website\Model\ProjectVersion;
+use Doctrine\Website\Tests\TestCase;
+
+use function assert;
+use function file_get_contents;
+use function is_dir;
+use function json_decode;
+
+class ProjectsTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        $buildDir = __DIR__ . '/../../../build-test';
+
+        if (is_dir($buildDir)) {
+            return;
+        }
+
+        self::markTestSkipped('This test requires ./bin/console build-website to have been ran.');
```

Comment:
```suggestion
        self::markTestSkipped('This test requires ./bin/console build-website to have been run.');
```

Reply:
I've also fixed the other occurrence where I copied the code from